recipes/gdal/post_3.5.0/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.15)
+project(gdal_cmake_wrapper)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+include(CMakePushCheckState)
+
+set(CMAKE_MODULE_PATH "${CMAKE_BINARY_DIR}")
+
+if (${GDAL_USE_Poppler})
+    find_package(poppler)
+    set(Poppler_VERSION_STRING ${poppler_VERSION})
+endif()
+
+file(GLOB CONAN_GENERATED_CMAKE_FILES "${CMAKE_BINARY_DIR}/Find*.cmake")
+foreach(CMAKE_FILE ${CONAN_GENERATED_CMAKE_FILES})
+    include(${CMAKE_FILE})
+endforeach()
```

Comment:
We should think to conan v2 migration, and this block won't fit in CMakeToolchain logic.

Reply:
removed, replaced with aliases to `CONAN_PKG::` targets and explicit find_package.

Reply:
so, since I failed at understanding how to migrate to V2, I tried to use aliases and find_packages, but this is a bit manual and I risk missing some.

I am not sure what is the best way to proceed.

I could manually write a list such as:
```
if (${GDAL_USE_PACKAGE})
  find_package(PACKAGE)
endif()
```
but that would be easy to make mistake and a bit painful to maintain.

I could try to automate the find_package(), but I do not know how to get the proper FindXXX.cmake given the conan package name.

I could switch to CMakeDeps, but I do not understand the logic of the new API and I did not find any example using it.

What do you suggest?